src/wp-includes/class-wp-autoload.php

```diff
@@ -0,0 +1,424 @@
+<?php
+/**
+ * Autoloader for WordPress classes.
+ *
+ * Include this file if you'd like to avoid having to create your own autoloader.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Autoloader.
+ */
+final class WP_Autoload {
+
+	/**
+	 * An array of classes and their respective paths.
+	 *
+	 * Note: The classnames should be lowercased.
+	 *
+	 * @access private
+	 *
+	 * @var array
+	 */
+	private static $classes_paths = array(
```

Comment:
This should use a class constant for performance reasons.

The way it is now, this array will always need to be allocated and assembled for every single request. With a class constant, it will just "exist" in memory once the class was retrieved from opcode cache.